zebra-test/src/command.rs

```diff
@@ -250,3 +257,24 @@ impl TestOutput {
         return self.output.status.code() == Some(1);
     }
 }
+
+/// Provide functions to manage a TempDir
+pub struct ZebraTestDir {
+    pub tempdir: Option<TempDir>,
+}
+
+impl ZebraTestDir {
+    pub fn new(prefix: &str) -> Self {
+        Self {
+            tempdir: Some(TempDir::new(prefix).unwrap()),
```

Comment:
`Result` has a helper for converting it into an option. I don't think we want to use it here, so if you already know this then feel free to ignore. In this case its probably better if we panic outright if the tempdir fails to create.

```rust
            tempdir: TempDir::new(prefix).ok(),
```